pittgoogle/pubsub.py

```diff
@@ -322,20 +323,42 @@ class Subscription:
             Schema name of the alerts in the subscription. Passed to :class:`pittgoogle.alert.Alert` for unpacking.
             If not provided, some properties of the Alert may not be available. For a list of schema names, see
             :meth:`pittgoogle.registry.Schemas.names`.
+        attribute_filter (str, optional):
+            Specify a filter to only receive the messages whose attributes match the filter. The filter is an immutable
+            property of a subscription. After you create a subscription, you cannot update the subscription to modify
+            the filter.
+        udf (str, optional):
+            Specify a JavaScript User-Defined Function (UDF). UDFs attached to a subscription can enable a wide range
+            of use cases, including: message filtering (based on the message payload and/or attributes), simple data
+            transformations, data masking and redaction, and data format conversions.
```

Comment:
You don't need to go into the details here, but do give the user a heads up that both of these strings must conform to very specific requirements and then provide links to the google documentation where they can learn what those requirements are.